### ISSUE TITLE
Super Robot Wars T

### ISSUE THREAD

**Issue by skrevenge** [skyline-emu/skyline-games-list#1177]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Tested Build

Edge - 6

### Android Version

11

### SoC type

Mediatek

### Device GSMArena link

https://www.gsmarena.com/xiaomi_redmi_note_8_pro-9812.php

### Game Behaviour

Triple buffering (on)
Executor 8
Texture hack (on)

- Runs without glitches or frame drop 
- It gives black screen after select protagonist. I've used a save, but it gets a black screen after finish the stage too.

https://user-images.githubusercontent.com/57811892/199629947-e5eaa7ac-47ba-4cc5-bc9e-d2b391247481.mp4

### Is Log Level is set to 'Info'?

- [X] Yes

### Log file

[SRW_T.txt](https://github.com/skyline-emu/skyline-games-list/files/9925029/SRW_T.txt)

**Comment by Berdugobugok** [comment]
I also add that it also gives black screen after the battle during intermission.

What i did for now is transferring  my save file to PC yuzu emulator and continue playing it there so i can bypass the black screen in that part.  Then once im on the battle map again, i copied the yuzu savefile and transferred to my skyline edge save file to resume my game there. Lmao 😂

**Comment by skrevenge** [comment]
> 

Yeah! i can do that too, but i'm trying to finish my tests without, or with a save that only skip an intro, or inicial cutscene. Looks like 30 don't need save, but he is crashing in my model. 😅

**Comment by SoulHate** [comment]
Any update on this? tried it on Edge V11 on SD 860 still getting black screen after character creation and after each stage :(

**Comment by juwaisri** [comment]
Still the same issues
